src/lightning_lite/plugins/collectives/__init__.py

```diff
@@ -0,0 +1,9 @@
+from lightning_lite.plugins.collectives.collective import Collective
+from lightning_lite.plugins.collectives.single_device_collective import SingleDeviceCollective
+from lightning_lite.plugins.collectives.torch_collective import TorchCollective
+
+__all__ = [
+    "Collective",
+    "TorchCollective",
+    "SingleDeviceCollective",
+]
```

Comment:
should we omit this as well for now to make it less discoverable?

Reply:
If an user is already importing `lite.plugins.collectives`, they've been discovered